benchexec/tools/nacpa.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchexec import result
+from benchexec.tools import template
+
+
+class Tool(template.BaseTool2):
+    """
+    This tool-info module runs nacpa, a parallel portfolio
+    of natively compiled CPAchecker instances.
+    """
+
+    REQUIRED_PATHS = ["bin", "lib"]
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("nacpa", subdir="bin")
+
+    def version(self, executable):
+        return self._version_from_tool(
+            executable, "--version", line_prefix="nacpa version"
+        )
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
+
+    def name(self):
+        return "nacpa"
+
```

Comment:
Could you also add a function `project_url()` that returns the tool's URL? This is quite useful and makes all tables link back to your tool page.

Reply:
Neat, I was not aware that this is what the project url is used for 😄